docs/source/examples/Ricci Tensor and Scalar Curvature symbolic calculation.ipynb

```diff
@@ -33,10 +34,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "syms = sympy.symbols(\"t chi theta phi\")\n",
-    "t, ch, th, ph = syms\n",
-    "m = sympy.diag(-1, cos(t) ** 2, cos(t) ** 2 * sinh(ch) ** 2, cos(t) ** 2 * sinh(ch) ** 2 * sin(th) ** 2).tolist()\n",
-    "metric = MetricTensor(m, syms)"
+    "AntiDeSitter().tensor()"
```

Comment:
```suggestion
    "metric = AntiDeSitter()"
```

Reply:
Also in the last few lines of notebook add a line `R.simplify()` before `R.expr`

Reply:
yes alright